app/pages/projects.cjsx

```diff
@@ -38,7 +35,7 @@ ProjectsPage = React.createClass
         delete query[key]
     newLocation = Object.assign {}, @props.location, {query}
     newLocation.search = ""
-    @props.history.replace newLocation
```

Comment:
I think it should. It's passed in only for route-level components, but it's still deprecated.